cpu/stm32l1/periph/i2c.c

```diff
@@ -97,9 +97,9 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
     NVIC_EnableIRQ(i2c_config[dev].er_irqn);
 
     /* configure pins */
-    gpio_init(i2c_config[dev].scl, GPIO_DIR_OUT, GPIO_PULLUP);
+    gpio_init(i2c_config[dev].scl, GPIO_OD_PU);
```

Comment:
This should be board dependent, some boards already have a pull-up on the I2C lines, which would reduce the pull-up to 5K instead of 10K.

Same goes for line 102.